packages/office-ui-fabric-react/src/components/Slider/__snapshots__/Slider.test.tsx.snap

```diff
@@ -2,18 +2,112 @@
 
 exports[`Slider renders Slider correctly 1`] = `
 <div
-  className="ms-Slider ms-Slider-enabled undefined ms-Slider-row undefined"
+  className=
+      ms-Slider
+      ms-Slider-enabled
+      ms-Slider-row
```

Comment:
Two undefineds in original seem to imply custom classNames.. I'd recommend modifying the test to supply them so they show up in the output

Reply:
Actually these undefined's were actually extraneos because they are actually toggles of each other (disabled vs enabled, etc).